### ISSUE TITLE
Conversion fails with Gotenberg 7.6

### ISSUE THREAD

**Issue by Limerick-gh** [paperless-ngx/paperless-ngx#2065]
### Description

Having Gotenberg 7.6 image with latest 1.10.0 version, resulting in 503 error while converting DOCX files. Switching back to Gotenberg 7.4 with exact same configuration works well. 
Is anyone facing same issue?

### Steps to reproduce

This is my current conf regarding Gotenberg:
`      PAPERLESS_TIKA_GOTENBERG_ENDPOINT: http://gotenberg:3000
      PAPERLESS_TIKA_ENDPOINT: http://tika:9998

  gotenberg:
    image: docker.io/gotenberg/gotenberg:7.6
    ports:
    - 3000:3000
    restart: unless-stopped
    command:
      - "gotenberg"
      - "--chromium-disable-routes=true"
      - "--api-timeout=300s"`

### Webserver logs

```bash
Log with Gotenberg 7.4:
[2022-11-29 19:46:23,945] [INFO] [paperless.management.consumer] Adding /usr/src/paperless/consume/DOC1.docx to the task queue.

[2022-11-29 19:46:24,078] [DEBUG] [paperless.barcodes] Detected mime type: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:46:24,080] [WARNING] [paperless.barcodes] Unsupported file format for barcode reader: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:46:24,182] [INFO] [paperless.consumer] Consuming DOC1.docx

[2022-11-29 19:46:24,184] [DEBUG] [paperless.consumer] Detected mime type: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:46:24,198] [DEBUG] [paperless.consumer] Parser: TikaDocumentParser

[2022-11-29 19:46:24,210] [DEBUG] [paperless.consumer] Parsing DOC1.docx...

[2022-11-29 19:46:24,210] [INFO] [paperless.parsing.tika] Sending /usr/src/paperless/consume/DOC1.docx to Tika server

[2022-11-29 19:46:24,444] [WARNING] [paperless.parsing.tika] Unable to extract date for document /usr/src/paperless/consume/DOC1.docx: 'Creation-Date'

[2022-11-29 19:46:24,446] [INFO] [paperless.parsing.tika] Converting /usr/src/paperless/consume/DOC1.docx to PDF as /tmp/paperless/paperless-4bew9qo0/convert.pdf

[2022-11-29 19:46:27,622] [DEBUG] [paperless.consumer] Generating thumbnail for DOC1.docx...

[2022-11-29 19:46:27,634] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient /tmp/paperless/paperless-4bew9qo0/convert.pdf[0] /tmp/paperless/paperless-4bew9qo0/convert.webp

[2022-11-29 19:47:13,730] [DEBUG] [paperless.consumer] Saving record to database

[2022-11-29 19:47:13,736] [DEBUG] [paperless.consumer] Creation date from st_mtime: 2022-11-29 19:45:32.779000+01:00

[2022-11-29 19:47:29,146] [INFO] [paperless.handlers] Assigning correspondent ABCD to 2022-11-29 DOC1

[2022-11-29 19:47:29,156] [INFO] [paperless.handlers] Assigning document type xyz to 2022-11-29 ABCD DOC1

[2022-11-29 19:47:34,027] [DEBUG] [paperless.consumer] Deleting file /usr/src/paperless/consume/DOC1.docx

[2022-11-29 19:47:34,062] [DEBUG] [paperless.parsing.tika] Deleting directory /tmp/paperless/paperless-4bew9qo0

[2022-11-29 19:47:34,066] [INFO] [paperless.consumer] Document 2022-11-29 ABCD DOC1 consumption finished


Log with Gotenberg 7.6:
[2022-11-29 19:51:41,143] [INFO] [paperless.management.consumer] Adding /usr/src/paperless/consume/DOC2.docx to the task queue.

[2022-11-29 19:51:41,463] [DEBUG] [paperless.barcodes] Detected mime type: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:51:41,465] [WARNING] [paperless.barcodes] Unsupported file format for barcode reader: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:51:41,540] [INFO] [paperless.consumer] Consuming DOC2.docx

[2022-11-29 19:51:41,542] [DEBUG] [paperless.consumer] Detected mime type: application/vnd.openxmlformats-officedocument.wordprocessingml.document

[2022-11-29 19:51:41,557] [DEBUG] [paperless.consumer] Parser: TikaDocumentParser

[2022-11-29 19:51:41,568] [DEBUG] [paperless.consumer] Parsing DOC2.docx...

[2022-11-29 19:51:41,569] [INFO] [paperless.parsing.tika] Sending /usr/src/paperless/consume/DOC2.docx to Tika server

[2022-11-29 19:51:41,717] [WARNING] [paperless.parsing.tika] Unable to extract date for document /usr/src/paperless/consume/DOC2.docx: 'Creation-Date'

[2022-11-29 19:51:41,718] [INFO] [paperless.parsing.tika] Converting /usr/src/paperless/consume/DOC2.docx to PDF as /tmp/paperless/paperless-qjpvtzgd/convert.pdf

[2022-11-29 19:51:51,818] [DEBUG] [paperless.parsing.tika] Deleting directory /tmp/paperless/paperless-qjpvtzgd

[2022-11-29 19:51:51,891] [ERROR] [paperless.consumer] Error while consuming document DOC2.docx: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert

Traceback (most recent call last):

  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 101, in convert_to_pdf

    response.raise_for_status()  # ensure we notice bad responses

  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status

    raise HTTPError(http_error_msg, response=self)

requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/consumer.py", line 337, in try_consume_file

    document_parser.parse(self.path, mime_type, self.filename)

  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 82, in parse

    self.archive_path = self.convert_to_pdf(document_path, file_name)

  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 103, in convert_to_pdf

    raise ParseError(

documents.parsers.ParseError: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert
```

### Paperless-ngx version

1.10.0

### Host OS

Raspberry Pi 4

### Installation method

Docker - official image

**Comment by stumpylog** [comment]
You could try with 7.7 as well.   Searching their issues, I see some related issues to DOCX files.

Our interface to it didn't change in the new version, so running 7.6 or 7.4 or 7.7 shouldn't matter

**Comment by stepman0** [comment]
I have the same issue, neither 7.6 nor 7.7 works (both 503), but 7.4 works.

**Comment by shamoon** [comment]
Seems like a problem with their project? Or perhaps your setup since a 503 to `http://gotenberg:3000/forms/libreoffice/convert` makes me think your gotenberg isnt up.

Worst case we can update the compose file to 7.4, but we have tests that run against 7.7 which seem to be passing OK: https://github.com/paperless-ngx/paperless-ngx/actions/runs/3560894605/jobs/5981302120#step:9:553

**Comment by Limerick-gh** [comment]
Thanks for pointing to newer version 7.7. This image does work in my case. So I will keep that. 👍

**Comment by mztiq** [comment]
I sadly still have the same error :(, here is my config regarding gotenberg/tika (installed via script):
```
      PAPERLESS_TIKA_ENABLED: 1
      PAPERLESS_TIKA_GOTENBERG_ENDPOINT: http://gotenberg:3000
      PAPERLESS_TIKA_ENDPOINT: http://tika:9998
```
Things I already tried:
- gotenberg V 7.4 and 7.7
  - Same error as with 7.6
- Changed gotenberg endpoint to http://gotenberg:3000/forms/libreoffice/convert#
  - Error changed to the following:
```
Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert/forms/libreoffice/convert
```
- Tried to expose port 3000 for the gotenberg service and changed gotenberg endpoint to http://<IP-ADDRESS>:3000/forms/libreoffice/convert#
  - Same error 503 
- Installed gotenberg as a separate docker container with port 3000 exposed and gotenberg endpoint to http://<IP-ADDRESS>:3000/forms/libreoffice/convert#
  - Same error 503 

Here's the error log of the webserver:
```
[2022-12-02 01:47:01,284] [INFO] [celery.worker.strategy] Task documents.tasks.consume_file[f03368b1-f476-4f49-aabc-91e44f67e00f] received
[2022-12-02 01:47:01,463] [INFO] [paperless.consumer] Consuming test.docx
[2022-12-02 01:47:01,507] [INFO] [paperless.parsing.tika] Sending /tmp/paperless/paperless-upload-x1q2ojf3 to Tika server
[2022-12-02 01:47:06,625] [WARNING] [paperless.parsing.tika] Unable to extract date for document /tmp/paperless/paperless-upload-x1q2ojf3: 'Creation-Date'
[2022-12-02 01:47:06,625] [INFO] [paperless.parsing.tika] Converting /tmp/paperless/paperless-upload-x1q2ojf3 to PDF as /tmp/paperless/paperless-5vbxnp5q/convert.pdf
[2022-12-02 01:47:16,989] [ERROR] [paperless.consumer] Error while consuming document test.docx: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert
Traceback (most recent call last):
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 101, in convert_to_pdf
    response.raise_for_status()  # ensure we notice bad responses
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/consumer.py", line 337, in try_consume_file
    document_parser.parse(self.path, mime_type, self.filename)
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 82, in parse
    self.archive_path = self.convert_to_pdf(document_path, file_name)
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 103, in convert_to_pdf
    raise ParseError(
documents.parsers.ParseError: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert
[2022-12-02 01:47:17,172] [ERROR] [celery.app.trace] Task documents.tasks.consume_file[f03368b1-f476-4f49-aabc-91e44f67e00f] raised unexpected: ConsumerError('test.docx: Error while consuming document test.docx: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert')
Traceback (most recent call last):
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 101, in convert_to_pdf
    response.raise_for_status()  # ensure we notice bad responses
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/asgiref/sync.py", line 280, in main_wrap
    raise exc_info[1]
  File "/usr/src/paperless/src/documents/consumer.py", line 337, in try_consume_file
    document_parser.parse(self.path, mime_type, self.filename)
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 82, in parse
    self.archive_path = self.convert_to_pdf(document_path, file_name)
  File "/usr/src/paperless/src/paperless_tika/parsers.py", line 103, in convert_to_pdf
    raise ParseError(
documents.parsers.ParseError: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/src/paperless/src/documents/tasks.py", line 177, in consume_file
    document = Consumer().try_consume_file(
  File "/usr/src/paperless/src/documents/consumer.py", line 356, in try_consume_file
    self._fail(
  File "/usr/src/paperless/src/documents/consumer.py", line 91, in _fail
    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception
documents.consumer.ConsumerError: test.docx: Error while consuming document test.docx: Error while converting document to PDF: 503 Server Error: Service Unavailable for url: http://gotenberg:3000/forms/libreoffice/convert
```

This is gotenbergs error log:
```
{"level":"error","ts":1669942036.9327428,"logger":"api","msg":"convert to PDF: lock long-running LibreOffice listener: start long-running LibreOffice listener: execute LibreOffice listener: context done: context deadline exceeded","trace":"b403b5e4-eee1-49af-8901-7ae6edfe6991","remote_ip":"172.18.0.6","host":"gotenberg:3000","uri":"/forms/libreoffice/convert","method":"POST","path":"/forms/libreoffice/convert","referer":"","user_agent":"python-requests/2.28.1","status":503,"latency":10171085350,"latency_human":"10.17108535s","bytes_in":47504,"bytes_out":19}
```

All containers are up and running:
```
docker-compose ps
        Name                       Command                  State                         Ports                   
------------------------------------------------------------------------------------------------------------------
paperless_broker_1      docker-entrypoint.sh redis ...   Up             6379/tcp                                  
paperless_db_1          docker-entrypoint.sh postgres    Up             5432/tcp                                  
paperless_gotenberg_1   /usr/bin/tini -- gotenberg ...   Up             3000/tcp                                  
paperless_tika_1        /bin/sh -c exec java -jar  ...   Up             9998/tcp                                  
paperless_webserver_1   /sbin/docker-entrypoint.sh ...   Up (healthy)   0.0.0.0:50056->8000/tcp,:::50056->8000/tcp
```

Any help would be appreciated, I'd really love to switch to paperless-ngx.

Thanks in advance.

**Comment by skwee357** [comment]
Just FYI. Having the same error on 7.6 and 7.7.
Switching to 7.4 works.
Let me know if I can provide any other helpful information.

**Comment by mztiq** [comment]
This is really weird, seems OP and you both found a workaround in using Gotenberg 7.4 but sadly thats not working for me.  
I already created a new issue (https://github.com/paperless-ngx/paperless-ngx/issues/2116) since this one has been closed. I got redirected to an open Gotenberg issue (https://github.com/gotenberg/gotenberg/issues/442) which has no solution yet.  

@skwee357 Just for the record: All you did was changing `image: docker.io/gotenberg/gotenberg:7.6` to `image: docker.io/gotenberg/gotenberg:7.4` in the `docker-compose.yml` right?

**Comment by skwee357** [comment]
@mztiq hey. Yes, I've just changed to `7.4` and did `docker compose pull`

**Comment by mztiq** [comment]
I just tried it out again with `/gotenberg:7.4`  and it worked.  
Last time I tested I just did a `docker-compose up -d --force-recreate` and now `docker-compose pull` up front so maybe it was my fault for not making sure `gotenberg` actually is running in `7.4`.  
Thanks for making me try again :D

**Comment by deddc23efb** [comment]
I was having this issue with gotenburg 7.6 as recommended in the current (as of my writing) docker-compose.yml files.  Updating the gotenburg version image to 7.7.2 fixed the issue for me.

**Comment by mztiq** [comment]
Yes, the fix from `gulnap/gotenberg:libreofficefix` has been implemented to v7.7.2 (see https://github.com/gotenberg/gotenberg/issues/442#issuecomment-1371099828). Sadly my homeserver died yesterday, that's why I couldn't test v7.7.2 yet but if you can confirm it's working maybe create a PR to change the docker-compose.yml files accordingly?

**Comment by ofsaleem** [comment]
not going to re-open this, but did want to mention 7.8 failed on a docx and a pptx but when i set the gotenberg version back to 7.7.2 it worked
